db_upgrade_script/mosip_certify/sql/0.9.1_to_0.10.0_upgrade.sql

```diff
@@ -0,0 +1,33 @@
+\c inji_certify
+
+CREATE TABLE svg_template (
```

Comment:
can we rename this to rendering template?
the type of image used here is svg, it can be changed as well right?

Reply:
with additional field designating the type of template

Reply:
These changes have to be merged in release branch also and there are templates and repositories associated with this table. Can these changes be taken up for the next release? @vharsh

Reply:
We're a bit late in designing this template as this change has also gone to `released` and `collab`, hence I suggest this to be taken up in next release if possible.